scripts/build/TestImpactAnalysis/test_impact/base_test_impact.py

```diff
@@ -123,20 +125,13 @@ def _parse_arguments_to_runtime(self, args, sequence_type, runtime_args):
         runtime_args.append(f"--sequence={sequence_type}")
         logger.info(f"Sequence type is set to '{sequence_type}'.")
 
-        if args.get('safe_mode'):
```

Comment:
Let's replace all string lits for args as named consts at the top of the file or class so we have an easy reference (also handy if they ever need to change).